### PR TITLE
Convert UrlIcon knobs and actions to controls / args

### LINKED ISSUE #13067: UrlIcon story: convert knobs and actions to controls / args
**Describe the bug**
As part of a push to convert all stories that use deprecated `@storybook/addon-knobs` and `@storybook/addon-actions` to use [controls](https://storybook.js.org/docs/react/essentials/controls) and [action argType annotation](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation)

Convert `UrlIcon` story in `ui/components/ui/url-icon/url-icon.stories.js` to use [controls](https://storybook.js.org/docs/react/essentials/controls) and [action argType annotation](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation)

Read the [MetaMask extension storybook documentation guidelines](https://metamask.github.io/metamask-storybook/?path=/story/getting-started-documentation-guidelines--page) to get a better understanding of writing stories. Providing MDX documentation although helpful, is a bonus and not a requirement for this issue.

To start storybook:
```
yarn storybook
```

**Requirements**
- [ ] Story has `argTypes` that align with component api / props
- [ ] All instances of `@storybook/addon-knobs` have been removed in favour of control args
- [ ] All instances of `@storybook/addon-actions` have been removed in favour of [action argType annotation](https://storybook.js.org/docs/react/essentials/actions#action-argtype-annotation)

Any further improvements to the story interaction or documentation is a bonus

**Images**

Screen Shot 2021-12-12 at 6 02 05 PM
Screen Shot 2021-12-12 at 6 03 11 PM

### PR DESCRIPTION
**Pull request by bestony**
## Explanation

Convert UrlIcon knobs and actions to controls / args

fixed #13067

## More information

* Fixes: #13067

## Screenshots/Screencaps

image

image

### CODE REVIEW

File: ui/components/ui/url-icon/url-icon.stories.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
 import React from 'react';
-import { text } from '@storybook/addon-knobs';
 import UrlIcon from './url-icon';
 
 export default {
   title: 'Components/UI/UrlIcon',
   id: __filename,
+  argType: {
+    name: { description: 'Icon Name', control: 'text' },
+    url: { description: 'Icon File URL', control: 'text' },
```
Comment (georgewrmarshall) [inline ui/components/ui/url-icon/url-icon.stories.js:9]: Let's keep the control names the same as the prop names. I think that should help educated devs on the component api quicker.

```suggestion
    name: { control: 'text' },
    url: { control: 'text' },
```

File: ui/components/ui/url-icon/url-icon.stories.js [hunk 2]
```diff
@@ -1,10 +1,25 @@
...
+  args: {
+    name: 'AST',
+    url: 'AST.png',
+    className: '',
+    fallbackClassName: '',
+  },
+};
+
+export const DefaultStory = (args) => {
+  return <UrlIcon name={args.name} url={args.url} />;
```
Comment (georgewrmarshall) [inline ui/components/ui/url-icon/url-icon.stories.js:22]: The `DefaultStory` should accept all available `argTypes`. I usually just spread the `...args` but there has been some concern about when we convert to typescript so we can just declare them here too.

```suggestion
  return <UrlIcon name={args.name} url={args.url} className={args.className} fallbackClassName={fallbackClassName} />;
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
You the best @bestony! Thanks for the contribution. Made a couple suggestions.

**Review by bestony — COMMENTED** [review]
done @georgewrmarshall

**Review by georgewrmarshall — DISMISSED** [review]
lgtm! Thanks again for the contribution! 💯

**Comment by bestony** [comment]
Hi,@ryanml, can you review my pull request?

**Review by darkwing — DISMISSED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Review by brad-decker — APPROVED** [review]

**Pull request merged**